e2e/node/e2e.test.ts

```diff
@@ -82,8 +82,8 @@ const TEST_USER_AGENT = `Node-based solid-client-access-grant end-to-end tests r
   process.env.CI === "true" ? "in CI" : "locally"
 }`;
 const addUserAgent =
-  (myFetch: typeof fetch, agent: string): typeof fetch =>
-  (input: Parameters<typeof fetch>[0], init: Parameters<typeof fetch>[1]) =>
```

Comment:
Why change the utility types into a duplicate of the underlying types?

Reply:
There was a type error because the underlying type was overloaded with one of the overloads not having a second parameter.

So the type error was the Parameter<type of fetch>[1] possibly indexing a const array of length 1.